.eslintignore

```diff
@@ -1,10 +1,7 @@
-build/*
-coverage/*
-dist/*
-public/*
-node_modules/*
-VETS-ONLY/*
-src/js/*
-fractalEntry.js
-vets-sketchapp/*
-/src/utils/*
+**/build/*
+**/dist/*
+**/public/*
+**/node_modules/*
+**/.cache/
+**/*.config.*
+**/*.conf.*
```

Comment:
not sure if we want to lint our webpack / karma configs.

Reply:
Probably not